src/bin/pgcopydb/catalog.c

```diff
@@ -2339,7 +2339,8 @@ catalog_lookup_s_table_by_name(DatabaseCatalog *catalog,
 		"           group by oid"
 		"         ) p "
 		"        on p.oid = t.oid"
-		"   where nspname = $1 and relname = $2 ";
+		"   where trim(nspname, '\"') = trim($1, '\"') "
+		"   and trim(relname, '\"') = trim($2, '\"') ";
```

Comment:
@dimitri I'm not sure how it scales well when we have lots of entries in catalog., Should we consider creating an index for nspname + relname?

Reply:
When using wal2json I believe we have the table OID available to us, we should probably use that. I didn't understand if the problem you're trying to solve is just in wal2json or both with test_decoding too?

Reply:
I'm trying to solve this for both wal2json & test_decoding.